src/Mod/TechDraw/Gui/Workbench.cpp

```diff
@@ -233,7 +233,7 @@ Gui::MenuItem* Workbench::setupMenuBar() const
 
     // views from other workbenches
     Gui::MenuItem* other = new Gui::MenuItem;
-    other->setCommand("Views From Other Workbenches");
+    other->setCommand("Views from Other Workbenches");

```

Comment:
```suggestion
    other->setCommand("Views From Other Workbenches");

```

Command in (context) menu + 4letter word acc. to guide (From is mentioned there)